web/templates/web/studies-list.html

```diff
@@ -73,7 +83,11 @@ <h1>{{study.name}}</h1>
                         {% if study.eligible_children %}
                         <p><strong>{{ study.eligible_children }} {% trans 'may be eligible.' %}</strong></p>
                         {% endif %}
-                        </small>
+                        {% if study.study_type.is_external %}
+                        <em>External study</em>
```

Comment:
Great idea - @okaycj , for the text under the "All studies" tab, could you please change it from:

"Please note you'll need a laptop or desktop computer (not a mobile device) running Chrome or Firefox to participate, unless a specific study says otherwise."

To:

"Lookit is growing! We are now showing links to outside studies along with those happening here on Lookit. Use the tabs above to see activities you can do right now, or scheduled activities you can sign up for.

Please note you'll need a laptop or desktop computer (not a mobile device) running Chrome or Firefox to participate, unless a specific study says otherwise."

Reply:
This copy update has been addressed in #831.  Thank you.